Order history, Templating, Payment and Price engine section update (#9)
ezsystems/commerce-documentation: docs/guide/order_history/orderhistory.md

# Order history

Order history enables you to see an overview and details of your orders or other documents from the ERP system.
You can see documents based on online purchases as well as orders placed e.g. by a telephone call.

> On the detail page you can see the buyer, delivery address, ordered items and some more details about the order,

Reviewer: ```suggestion
On the detail page you can see the buyer, delivery address, ordered items and other details about the order,
```
